README.md

```diff
@@ -123,6 +123,8 @@ Makefile.
 	$ vi Makefile            # ..or..
 	$ vi Makefile.conf
 
+Initialize the `abc` submodule with `git submodule update --init`.
```

Comment:
You also need to update the submodule each time you pull, so this isn't enough.

Reply:
How about "After cloning or pulling, intialize the `abc` submodule with `git submodule update --init`?